### ISSUE TITLE
its crashes for me , not sure if i done something wrong 

### ISSUE THREAD

**Issue by 6mas** [d0p3t/SubNotifier#12]
> twitch-subnotifier@1.0.3 start C:\Users\james\Desktop\SubNotifier-master
> nodemon src/index.js --exec babel-node --presets es2015,stage-2

[nodemon] 1.17.5
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `babel-node src/index.js --presets es2015,stage-2`
2:14:28 PM - info: STARTING SubNotifier...
C:\Users\james\Desktop\SubNotifier-master\node_modules\private\private.js:72
  for (var names = originalGetOPNs(object),
                   ^

TypeError: Cannot convert undefined or null to object
    at getOwnPropertyNames (<anonymous>)
    at Function.getOwnPropertyNames (C:\Users\james\Desktop\SubNotifier-master\node_modules\private\private.js:72:20)
    at TwitchBot.Start (C:/Users/james/Desktop/SubNotifier-master/src/lib/TwitchBot.js:82:32)
    at Object.<anonymous> (C:/Users/james/Desktop/SubNotifier-master/src/index.js:26:5)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at loader (C:\Users\james\Desktop\SubNotifier-master\node_modules\babel-register\lib\node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (C:\Users\james\Desktop\SubNotifier-master\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
[nodemon] app crashed - waiting for file changes before starting...

**Comment by d0p3t** [comment]
Could you give me complete steps to reproduce this issue?

We shortly talked and I saw you had outdated `node` and `npm` versions. If you could fix that and then try again, that would be great.

I am currently unable to reproduce this issue.

Perhaps your config file is incorrect. I have planned to eventually rewrite the config file anyway. The parsing isn't the greatest.

**Comment by 6mas** [comment]
yea i just updated it there and it works now  nice one man <3 , must of been out of date like you said , thanks for the reply

**Comment by 6mas** [comment]
ok it crashed when someone gifted a sub , https://gyazo.com/45e0a92ad2f11d75f39c44df032fa53a

**Comment by d0p3t** [comment]
Thank you for this. I will work on a fix right now

**Comment by 6mas** [comment]
thanks man

**Comment by 6mas** [comment]
> Could you give me the output of the debug if you enable it in your config.js
enableDebug: true,
And also make sure you have at least 3 custom gift subscription messages (another issue with the 
parsing)``

i fix that  with "  this._messagesCount[channel] = { subscriptions: 0, resubscriptions: 0, bits: 0, giftsubscriptions: 0 };" , for some reason that didnt edit when i download the new folders , i noticed it in your code updates ,  not sure if that was the cause , but it doesnt crash when i changed it
